q2_diversity/plugin_setup.py

```diff
@@ -240,6 +240,71 @@
     citations=[citations['legendrelegendre']]
 )
 
+plugin.methods.register_function(
+    function=q2_diversity.tsne,
+    inputs={'distance_matrix': DistanceMatrix},
+    parameters={
+        'number_of_dimensions': Int % Range(1, None),
```

Comment:
Is 1 a valid value for this method? The implementation code doesn't handle the case where 1 is specified, and I imagine anything less than 2 doesn't make much sense. If that is correct, please update this parameter:

```suggestion
        'number_of_dimensions': Int % Range(2, None),
```